README.md

```diff
@@ -0,0 +1,3 @@
+[![Build Status](https://travis-ci.org/newdevtech/newdev_api.svg?branch=develop)](https://travis-ci.org/newdevtech/newdev_api)
+
```

Comment:
Please remove this line.

Reply:
Isn't that the build status?

Reply:
The empty line, I mean.

Reply:
We need to have the badges on the same horizontal line.